features/fxa-migration/README.md

```diff
@@ -0,0 +1,38 @@
+
+AMO to Firefox Account Migration UX Specs
+=========================================
+
+So that I can reduce the numbers of logins I have, as a Mozilla Add-ons user, I want to switch from using my AMO login and start using a Firefox Account.
+
+## User Flow Diagram
+Red line denotes common post-migration login experience.
+![User Flow Diagram](https://www.lucidchart.com/publicSegments/view/c45f10ff-5084-407d-8322-cbc8b6cd25be/image.png)
+
+## Migrating from AMO
+* When a currently authenticated AMO user returns to AMO, they will see a notice prompting them to upgrade their account.
+* When a currently authenticated AMO user chooses the call to action in the notice, they are taken to the AMO login to confirm their identity.
+* When an existing AMO user enters their email address, they are taken to the AMO login.
+* When they log in successfully, they are shown a notice prompting them to upgrade their account.
+* When they click **Continue** and do not have a Firefox Account, they taken to FxA registration that is pre-populated with their email.
+* When they click **Continue** and do have a Firefox Account, they taken to FxA login that is pre-populated with their email.
+* When they click **Skip**, they taken to AMO which contains an upgrade notice.
+* When they attempt to use an email that is unverified with FxA, they are taken to the confirm your account screen.
```

Comment:
There's a 3-way decision in the flowchart before login happens but I think we need them to login to FxA regardless. I think this might be out of scope because that's already part of the standard FxA flow. The user would see this after registering for FxA as well but that isn't mentioned in the flowchart (and likewise I don't think it's needed in there).